README.md

```diff
@@ -53,6 +53,74 @@ loader.show(options); // options is optional
 loader.hide();
 ```
 
+##  Angular 2 Example
+
+#### main.ts
+```js
+import { LoadingIndicator } from "nativescript-loading-indicator"; // import Loading inidicator here
+
+@NgModule({
+  declarations: [
+    AppComponent
+  ],
+  bootstrap: [
+    AppComponent
+  ],
+  imports: [
+    HttpModule,
+    NativeScriptModule
+  ],
+  providers: [
+    LoadingIndicator // Add "LoadingIndicator" on Angular 2 providers
```

Comment:
This will error since `LoadingIndicator` is just a class with no decorated metadata; it would need the `Injectable` decorator for this to work. Confused at this PR?